### ISSUE TITLE
questions about the procedure of test()

### ISSUE THREAD

**Issue by jacobwei** [ganeshjawahar/mem_absa#4]
In the method "test" in model.py, it seems that there is a module about training the model using the test data.(from line 236 to 244 ).
I wonder why we should use test golden label here and in line 230, is it used as a validation ?
I'm new in tensorflow, please pardon me if there were errors in my understanding.

**Comment by ganeshjawahar** [comment]
Yes, its a bug. Change that portion to this:
`loss = self.sess.run([self.loss],
                                        feed_dict={
                                            self.input: x,
                                            self.time: time,
                                            self.target: target,
                                            self.context: context})`
so that we just seek the loss without doing backprop.

Also there is a bug in data.py. We need to discard input samples from the class label: 'conflict' as suggested by the author.

**Comment by dyngts** [comment]
Hi @ganeshjawahar,

can you elaborate more about bug in data.py ? What should i change? Thank you!
